lib/presentation/root/main_navigation_screen.dart

```diff
@@ -191,7 +194,6 @@ class MainNavigationScreen extends StatelessWidget {
               backgroundColor: Colors.white,
               color: const Color(0xFF576760),
```

Comment:
Extract colors to a color utility file or as extensions on the Colors class. Let's minimize hardcoding colors as much as possible in the presentation layer